core/server/api_container/server/startosis_engine/kurtosis_instruction/wait/wait.go

```diff
@@ -31,7 +31,7 @@ const (
 	TimeoutArgName     = "timeout"
 
 	defaultInterval = 1 * time.Second
-	defaultTimeout  = 15 * time.Minute
+	defaultTimeout  = 10 * time.Second
```

Comment:
Can we pass in a time out to the eth package then?

Reply:
https://github.com/kurtosis-tech/eth2-package/pull/78